### PR TITLE
markdown supports

### PR DESCRIPTION
**Pull request by CSLMeyers**
markdown supports heading/imagelink/delete/bolditalic styles

### CODE REVIEW

File: dist/Components/Basic/Markdown/Markdown.d.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export declare class Markdown extends React.Component<IProps> {
     private renderListBullet;
     private renderLink;
     private renderText;
+    private renderLinkImage;
```
Comment (DongyuZhao) [inline dist/Components/Basic/Markdown/Markdown.d.ts]: why link image?

File: dist/Components/Basic/Markdown/MarkdownTypes.d.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+export interface TextData {
+    type: MarkdownTypes;
+    data: any;
+    style?: StyleProp<TextStyle>;
+}
+export interface LinkData extends TextData {
+    link: string;
+}
+export interface ImageLinkData extends LinkData {
```
Comment (DongyuZhao) [inline dist/Components/Basic/Markdown/MarkdownTypes.d.ts]: ImageLink or Image

File: src/Components/Basic/Markdown/Styles.ts [hunk 3]
```diff
@@ -4,12 +4,37 @@ import {
...
+    h2: {
+        fontSize: 24,
+    } as TextStyle,
+    h3: {
+        fontSize: 18,
+    } as TextStyle,
+    h4: {
+        fontSize: 16,
+    } as TextStyle,
+    h5: {
```
Comment (faceach) [inline src/Components/Basic/Markdown/Styles.ts]: Please check whether Markdown config was included in card config.

File: src/Components/Basic/Markdown/MarkdownTypes.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+};
+
+export interface MarkdownFormatter {
+    type: MarkdownTypes;
+    pattern: any;
+    isSimpleText: boolean;
+}
+
+export const Markdown_Formatter_Config: MarkdownFormatter[] = [
+    {
```
Comment (faceach) [inline src/Components/Basic/Markdown/MarkdownTypes.ts]: could you list comments on each pattern?

File: src/Components/Basic/Markdown/FitImage.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import {
+    Image,
+} from 'react-native';
+
+interface IProps {
```
Comment (faceach) [inline src/Components/Basic/Markdown/FitImage.tsx]: Should there be a maxWidth or maxHeight props here?

File: src/Components/Basic/Markdown/Styles.ts (hunk 3 above)
Comment (CSLMeyers) [reply src/Components/Basic/Markdown/Styles.ts]: there is no config about Markdown before.

**Pull request merged**